var/spack/repos/builtin/packages/vtk/package.py

```diff
@@ -124,8 +124,8 @@ class Vtk(CMakePackage):
     patch("vtk_movie_link_ogg.patch", when="@8.2")
     patch("vtk_use_sqlite_name_vtk_expects.patch", when="@8.2")
     patch("vtk_proj_include_no_strict.patch", when="@9: platform=windows")
-    patch("vtk_alias_hdf5.patch", when="@9: platform=windows")
-    patch("vtk_findproj_config.patch", when="platform=windows")
+    patch("vtk_findproj_config.patch")
+    patch("vtk_alias_hdf5.patch", when="@9:")
```

Comment:
Can you add a brief comment above the `patch` directive explaining:
1. What is the effect of the patch
2. Why it is needed
3. Where it was taken from

?

Reply:
I wrote the patches from scratch, so there's no source to point to outside of the git history, but I added everything else requested.